src/main/resources/client_secret.json

```diff
@@ -1 +1,14 @@
-{"installed":{"client_id":"436986732241-nvjfkarirkhmcq7m3r1uemdau518uvdo.apps.googleusercontent.com","project_id":"terra-cli-dev","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"w7TLNFm2fog61d2cFNC_fO5L","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}
\ No newline at end of file
+{
+  "installed": {
```

Comment:
There was a problem initially with the JSON parsing inside the Google OAuth library, and I pretty-printed this file in an effort to sanity check it.